_posts/2016-05-17-foreman-community-newsletter-may-2016.md

```diff
@@ -0,0 +1,134 @@
+[rfcs]: https://github.com/theforeman/rfcs
+
+### Foreman Events Page
+
+The Foreman website now has it's own calendar! You can head to
+[http://theforeman/events][/events] to see all the upcoming things happening in
+& around Foreman. This page also has an ICS link which you can add to your own
+calendar system to automatically keep up-to-date with new events.
+
+The events system is powered by a simple YAML file in our [Github
+repo][events-yaml] so if you're attending or organising a Foreman event, or
+speaking about Foreman somewhere, do send us a patch to add your event!
+
+[events-yaml]: https://github.com/theforeman/theforeman.org/blob/gh-pages/_data/events.yml
+
+### NETWAYS Foreman Training Course donated
+
+The lovely folks at NETWAYS have decided to release their Foreman training
+course to the Foreman community! You can find more details on how it happened
+[here][foremanblog] and [here][netwaysblog], and the actual training course
+[here][trainingcourse]. Thanks NETWAYS!
+
+[foremanblog]: http://theforeman.org/2016/05/new-foreman-training-course.html
+[netwaysblog]: https://blog.netways.de/2016/04/29/foreman-training-release
+[trainingcourse]: http://netways.github.io/foreman-training/
+
+### Katello 3.0 RC4
+
+The Katello 3.0 release train is rolling on, and we're now up to RC4, with RC5
+likely before the final release. As ever, if you have spare time to road-test
+our release candidates, then the increased quality of the release is hugely
+appreciated by the whole community. You can find out more at the
+[RC4 release thread][katello-rc4]. Thanks for your help!
+
+[katello-rc4]: https://groups.google.com/d/topic/foreman-users/AYOBldNctN8/discussion
+
+### New media and blogs
+
+#### [Community demo](https://youtu.be/-bCIcN5i-24) on 28th April 2016
+
+We held another community demo this month, in which we covered the usual
+community banter from me, some Remote Execution and Libvirt updates from
+Stephen and Lukas, and a lengthy chat with Dirk from NETWAYS regarding the new
+Foreman Community training course.
+
+#### [Case Study on Discovery and Custom UIs with MyHeritage](https://youtu.be/bBZzoLM7fDM)
+
+Another Case Study happened this month, and this time we chatted with Imri and
+Elad from MyHeratge about the additions they've contributed to Foreman
+Discovery, how they modified the UI to make a simple developer portal for
+launching VMs, and more. Thanks guys!
+
+#### [Deep Dive on Recent Smart Proxy updates](https://www.youtu.be/Bwv8nU_-L5c)
+
+In another of our dives into the inner workings of Foreman, Dmitri Dolguikh
+(witlessb) gave us a look at recent changes to the smart-proxy core, using the
+changes to the puppet_proxy module needed for Puppet 4 support as an illustration.
+
+#### [Automating an R&D lab - Julien Pivotto, OSDC](https://www.youtu.be/csxeyyzZPvo)
+
+Julien gave a great talk about how to use Foreman for automating an entire lab
+at OSDC - this is definitely worth watching. Thanks Julien!
+
```

Comment:
Satellite 6.2 beta release might be also interesting for upstream users.

https://access.redhat.com/blogs/1169563/posts/2280311